test/objectwrap.cc

```diff
@@ -172,11 +172,30 @@ class Test : public Napi::ObjectWrap<Test> {
 
 std::string Test::s_staticMethodText;
 
+#ifdef NAPI_CPP_EXCEPTIONS
```

Comment:
We should perform this test in both cases – if C++ exceptions are enabled, *as well as* if they are not enabled.